.github/workflows/lint-participant-manager-datastore.yml

```diff
@@ -0,0 +1,27 @@
+name: lint-participant-manager-datastore
```

Comment:
Please add comment that describes what this file is and how it operates.

Reply:
Added comment in `lint-participant-manager-datastore.yml`.
This file is used for adding styleCheck for participant manager and participant manager datastore.
[Issue-2491](https://github.com/GoogleCloudPlatform/fda-mystudies/issues/2491)